torch_ort_inference/torch_ort/ortinferencemodule/provider_options.py

```diff
@@ -0,0 +1,29 @@
+class ProviderOptions:
+    def __init__(
+        self,
+        provider=""
```

Comment:
Could you please make provider a required field?

Reply:
Noted. PR updated.